test/extended/builds/s2i_extended_build.go

```diff
@@ -87,28 +95,26 @@ var _ = g.Describe("[builds][Slow] s2i extended build", func() {
 		oc.SetOutputDir(exutil.TestContext.OutputDir)
 
 		g.It("should use assemble-runtime script from URL", func() {
-			const buildConfigName = "java-extended-build-from-url"
-			const buildName = buildConfigName + "-1"
 
-			g.By("creating build config")
-			err := oc.Run("create").Args("-f", scriptsFromUrlBc).Execute()
+			g.By("creating jvm-runner configuration")
+			err := exutil.CreateResource(runnerConf, oc)
 			o.Expect(err).NotTo(o.HaveOccurred())
 
-			g.By("running the build")
-			out, err := oc.Run("start-build").Args("--build-loglevel=5", buildConfigName).Output()
-			if err != nil {
-				fmt.Fprintf(g.GinkgoWriter, "\nstart-build output:\n%s\n", out)
-			}
+			g.By("building jvm-runner image")
+			exutil.StartBuild("jvm-runner", oc)
+			err = exutil.WaitForBuild(oc.Namespace(), "jvm-runner", "jvm-runner-1", oc)
```

Comment:
Yes, I also thought about it, but decided that separation between tests is more important in this case. Let's keep it as-is.